spk/tcl/Makefile

```diff
@@ -1,29 +1,30 @@
-SPK_NAME = tcl
-SPK_VERS = 8.6.6
-SPK_REV = 1
-SPK_ICON = src/tcl.png
+SPK_NAME	= tcl
+SPK_VERS	= 8.6.6
+SPK_REV		= 6
+SPK_ICON	= src/tcl.png
 
-DEPENDS = cross/$(SPK_NAME)
+DEPENDS		= cross/$(SPK_NAME)
 
 MAINTAINER	= Tcl developer Xchange
```

Comment:
Please check https://github.com/SynoCommunity/spksrc/wiki/Makefile-variables and correct MAINTAINER, MAINTAINER_URL.